api/v1/helpers/verbs/doPut.js

```diff
@@ -47,39 +81,35 @@ function doPut(req, res, next, props) {
     putPromise = u.getUserNameFromToken(req)
       .then((user) => redisModelSample.putSample(req.swagger.params, user));
   } else {
+    let instance;
     const puttableFields =
       req.swagger.params.queryBody.schema.schema.properties;
     putPromise = u.findByKey(
         props, req.swagger.params
       )
       .then((o) => u.isWritable(req, o))
       .then((o) => {
-        const keys = Object.keys(puttableFields);
-        for (let i = 0; i < keys.length; i++) {
-          const key = keys[i];
-          if (toPut[key] === undefined) {
-            let nullish = null;
-            if (puttableFields[key].type === 'boolean') {
-              nullish = false;
-            } else if (puttableFields[key].enum) {
-              nullish = puttableFields[key].default;
-            }
-
-            o.set(key, nullish);
+        if (props.modelName === 'Generator') {
```

Comment:
same comment here. Do you want to move it to the generator controller ?

Reply:
let's leave it for another pull request